src/Giraffe/HttpHandlers.fs

```diff
@@ -449,4 +449,9 @@ let negotiate (responseObj : obj) =
                 |> text)
             <| ctx)
         // response object
-        responseObj
\ No newline at end of file
+        responseObj
+///Rdirect to route with 302 status
```

Comment:
Typo: Redirect
Also please put a space between the previous handler. thank you!